frontend/src/components/LaunchForm.tsx

```diff
@@ -0,0 +1,131 @@
+{
+    const classes = useStyles();
+
+    const changeAddress =  (event : React.ChangeEvent<HTMLInputElement>) => {
+        setAddress(event.target.value.length && event.target.value[0].trim() === '' ? 
+            event.target.value.slice(1) : event.target.value);
+    }
+
+    const changePort = (event : React.ChangeEvent<HTMLInputElement>)  => {
+        setPort(event.target.value.length && event.target.value[0].trim() === '' ? 
+            event.target.value.slice(1) : event.target.value);
+    }
+
+    const changeLogin =  (event : React.ChangeEvent<HTMLInputElement>) => {
+        setLogin(event.target.value.length && event.target.value[0].trim() === '' ? 
+            event.target.value.slice(1) : event.target.value);
+    }
+
+    const changePassword = (event : React.ChangeEvent<HTMLInputElement>)  => {
+        setPassword(event.target.value.length && event.target.value[0].trim() === '' ? 
+            event.target.value.slice(1) : event.target.value);
+    }
+
+    const submitForm = (e : any) => {
+        e.preventDefault();
+
+        if (adress === "")
+            setAddress(DEFUALT_ADDRESS);
+
+        if (port == "")
+            setPort(DEFAULT_PORT);
+
+        //console.log(adress, port, login, password)
+        setFormPassed(true);
+    }
+
+    return <div className={classes.blurredBackground}>
+        <form className={classes.content} onSubmit={submitForm} >
+            <div className={classes.element}>
+                <div>Address:</div>
+                <input
+                    defaultValue=""
+                    value={adress}
+                    onChange={changeAddress}
+                    placeholder={DEFUALT_ADDRESS}
+                />
+            </div>
+
+            <div className={classes.element}>
+                <div>Port:</div>
+                <input
+                    defaultValue=""
+                    value={port}
+                    onChange={changePort}
+                />
+            </div>
+
+            <div className={classes.element}>
+                <div>Username:</div>
+                <input
+                    defaultValue=""
+                    value={login}
+                    onChange={changeLogin}
+                />
+            </div>
+
+            <div className={classes.element}>
+                <div>Password:</div>
+                <input
+                    defaultValue=""
+                    value={password}
+                    onChange={changePassword}
+                    type="password"
+                />
+            </div>
+
+            <button>
+                Connect
+            </button>
+        </form>
+    </div>
+}
+
+const useStyles = makeStyles(theme => ({
+    blurredBackground: {
+        position: "absolute",
+        width: "100%",
+        height: "100%",
+        backgroundSize: "cover",
+        backgroundColor: "#161616",
+        opacity:"0.8",
```

Comment:
imo ta przezroczystość jest bardziej wkurzająca niż pomocna.

Oraz powiększyłbym trochę ten obiekt

Reply:
ale to jest opacity na dodatkowy obiekt który zasłania panel zeby najpierw trzeba sie było zalogować to ma byc taki styl "zablokowania"